packages/mui-material/src/InputBase/InputBase.js

```diff
@@ -533,7 +533,6 @@ const InputBase = React.forwardRef(function InputBase(inProps, ref) {
         {startAdornment}
         <FormControlContext.Provider value={null}>
           <Input
-            ownerState={ownerState}
```

Comment:
This shouldn't have been here; it's covered in line 557 below only for non-host components.